examples/audio_classifier/raspberry_pi/README.md

```diff
@@ -0,0 +1,58 @@
+# MediaPipe Audio classifier example with Raspberry Pi
+
+This example uses [MediaPipe](https://github.com/google/mediapipe) with Python on
+a Raspberry Pi to perform real-time audio classification using audio streamed
+from the microphone.
+
+## Set up your hardware
+
+Before you begin, you need to
+[set up your Raspberry Pi](https://projects.raspberrypi.org/en/projects/raspberry-pi-setting-up)
+with Raspberry Pi OS (preferably updated to Buster).
+
+Raspberry Pi doesn't have a microphone integrated on its board, so you need to
+plug in a USB microphone to record audio.
+
+## Install PortAudio and MediaPipe
+
+This project requires the installation of the PortAudio library and MediaPipe for running inference.
+
+You can install these dependencies by using the provided script.
+
+```
+sh setup.sh
```

Comment:
We're already showing `sh setup.sh` in the step-by-step portion of this readme, so let's remove it from here. Just say 'You can install the required dependencies using the setup.sh script provided with this project.'

Reply:
Fixed it.